hazelcast-tpc-engine/src/main/java/com/hazelcast/internal/tpc/nio/NioAsyncSocket.java

```diff
@@ -268,16 +269,6 @@ public void flush() {
         }
     }
 
-    private void resetFlushed() {
```

Comment:
Can you restore the original code. This function is part of the NioAsyncSocket; not of the handler.

Reply:
Reverted https://github.com/hazelcast/hazelcast/pull/23740/commits/7483a2aca1f78971e2c5efd0d99d1d8b2dcbd201